ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -190,9 +190,39 @@ var _ = Context("Inside the default namespace", func() {
 				time.Second*15, time.Millisecond*500).Should(BeNil(), "autoscaler RoleBinding = %v", rbName)
 		})
 
+		It("should be able to update all Pods to Running", func() {
+			for _, workerPod := range workerPods.Items {
```

Comment:
Would you mind adding comments to explain the reason for updating Pod statuses manually?

Reply:
done